test/support/test_config.h

```diff
@@ -106,4 +106,10 @@
 
 #define TEST_HAS_NO_INT128
 
+#if defined(_MSC_VER) && __cplusplus <= 201703L
+#    define _MSVC_STL_CPP17_COMPLEX_CONSTEXPR_FNC_BROKEN 1

```

Comment:
Please consider:
`define _MSVC_STL_CPP17_COMPLEX_CONSTEXPR_FNC_BROKEN (defined(_MSC_VER) && __cplusplus <= 201703L)`

Please check if it works

Reply:
I checked and use this variant:
#define _PSTL_MSVC_CPP17_COMPLEX_CONSTEXPR_FNC_BROKEN (_MSC_VER && __cplusplus <= 201703L)